packages/params/src/whitelisted-entities/paymasters.ts

```diff
@@ -137,5 +137,8 @@ export const WhitelistedPaymasters: IWhitelistedEntity = {
   ],
 
   // Mantle testnet
-  5001: [getAddress("0x6Ea25cbb60360243E871dD935225A293a78704a8")],
+  5001: [
```

Comment:
Can we add the mainnet ones as well? @vignesha22 please provide the addresses to @0xSulpiride

Reply:
@0xSulpiride this is deployed in mainnet only not on testnet

Reply:
Please change it to Mainnet

Reply:
changed to mainnet